code/modules/client/preferences.dm

```diff
@@ -631,6 +631,8 @@ GLOBAL_LIST_INIT(bgstate_options, list(
 					</b> <a href='?_src_=prefs;preference=toggle_prefs;flag=[TOGGLE_MIDDLE_MOUSE_SWAP_HANDS]'><b>[toggle_prefs & TOGGLE_MIDDLE_MOUSE_SWAP_HANDS ? "On" : "Off"]</b></a><br>"
 			dat += "<b>Toggle Vendors Vending to Hands: \
 					</b> <a href='?_src_=prefs;preference=toggle_prefs;flag=[TOGGLE_VEND_ITEM_TO_HAND]'><b>[toggle_prefs & TOGGLE_VEND_ITEM_TO_HAND ? "On" : "Off"]</b></a><br>"
+			dat += "<b>Toggle Semi-Auto Ammo Display: \
```

Comment:
```suggestion
			dat += "<b>Toggle Semi-Auto Ammo Display Limiter: \
```
Or `Type` its a bit ambiguous at the moment.

![image](https://github.com/cmss13-devs/cmss13/assets/76988376/04a6f2e4-62e4-41c5-82a1-a29f4a2810cc)